map/search_mark.cpp

```diff
@@ -241,6 +245,16 @@ class SearchMarkTypeChecker
       {{"leisure", "sports_hall"},   SearchMarkType::Pitch},
       {{"leisure", "swimming_pool"}, SearchMarkType::Swimming},
       {{"leisure", "water_park"},    SearchMarkType::Swimming},
+      {{"amenity", "drinking_water"},                       SearchMarkType::DrinkingWater},
+      {{"amenity", "water_point"},                          SearchMarkType::DrinkingWater},
+      {{"man_made", "water_tap"},                           SearchMarkType::DrinkingWater},
+      {{"man_made", "water_well"},                          SearchMarkType::DrinkingWater},
+      {{"natural", "spring"},                               SearchMarkType::DrinkingWater},
+      {{"natural", "hot_spring"},                           SearchMarkType::DrinkingWater},
```

Comment:
Probably should be displayed as not drinking?
Or maybe should be removed from the "water" search altogether? (IDK why do we have it there..)

Reply:
Hot springs currently use the drinking water icon on the map. That's why I added it here. We may want to add a different map icon for them in a separate PR

https://github.com/organicmaps/organicmaps/blob/3e6eb90be6b06c05103b34dace98cb368b6ffc02/data/styles/default/include/Icons.mapcss#L174-L179